images/linux/scripts/installers/1804/basic.sh

```diff
@@ -97,6 +97,9 @@ apt-get install -y --no-install-recommends xvfb
 echo "Install libgtk"
 apt-get install -y --no-install-recommends libgtk-3-0
 
+echo "Install tk"
+apt install -y tk
```

Comment:
do we also need `--no-install-recommends` here as we do for other tools? also, I see that `--no-install-recommends` is used for 1604 as well

Reply:
I didn't found the option  '--no-install-recommends' in apt man page (as far a I know apt it's a different tool then apt-get).
Also we used the same command in building Ubuntu HostedToolCache.Python, and this build was was without any errors

Reply:
oh, it's `apt`, not `apt-get`, didn't notice that. Thanks for clarification.